gateway-discovery-cm/src/main/java/org/apache/knox/gateway/topology/discovery/cm/ClouderaManagerServiceDiscovery.java

```diff
@@ -356,6 +361,14 @@ private List<ApiService> getClusterServices(ServiceDiscoveryConfig serviceDiscov
         final ApiServiceList serviceList = servicesResourceApi.readServices(serviceDiscoveryConfig.getCluster(), VIEW_SUMMARY);
         services = serviceList == null ? new ArrayList<>() : serviceList.getItems();
 
+        services = services.stream().filter(service -> {
+          if (excludedServiceTypes.contains(service.getType())) {
```

Comment:
I'm wondering if we can avoid case-sensitivity issues with this. For example, if the case of the service type is mistyped in the config or changes on the CM side. Perhaps, we can lowercase the values prior to the comparison?

Reply:
This is a good idea. Let me fix that and submit a new PR soon.